src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2662,6 +2662,11 @@ export class StudyViewPageStore {
         default: []
     });
 
+    @autobind
+    public async getGenePanelData(genePanelId: string): Promise<GenePanel> {
+        return Promise.resolve(await defaultClient.getGenePanelUsingGET({genePanelId: genePanelId}));
```

Comment:
this is equivalent to just returning `defaultClient....` , doesnt need to be `async`